src/Application/Service/MocoService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Ph3\DockerArch\Application\Service;
+
+use Ph3\DockerArch\Domain\Service\Model\AbstractService;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Alexis NIVON <anivon@alexisnivon.fr>
+ */
+class MocoService extends AbstractService
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getOptionsResolver(): Options
+    {
+        $resolver = new OptionsResolver();
+
+        $resolver->setRequired('mock_filename');
+        $resolver->setAllowedTypes('mock_filename', 'string');
```

Comment:
In the future, why not handle multiple mock files. Or keep an single mock container approach.
TBD

Reply:
Good idea! 👍

Reply:
What about using this:

``` json
[
    {
        "include" : "file1.json"
    },
    {
        "include" : "file2.json"
    }
]
```

Seems to be implemented for this.

https://github.com/dreamhead/moco/blob/master/moco-doc/global-settings.md

Reply:
In fact, I doesn't think to multiple files for one "service" (one port) but I think about multiple files, each representing an independent service on a specific port.

Reply:
OK! My bad, I didn't understand your point of view!